tox.ini

```diff
@@ -1,21 +1,35 @@
 [tox]
 envlist =
+    flake8,
     py27,
     py37,
     py38,
+    coverage,
 
 [testenv]
 usedevelop = true
 commands =
-    py.test src -o python_files=test_jasmine.py --flake8 {posargs}
+    py.test []
```

Comment:
Is the restriction to `test_jasmine.py` not necessary any more?

Reply:
Does not seem to as it is the only test and coverage has gone up.